pkg/licenseapi/features.go

```diff
@@ -85,5 +86,6 @@ func GetVClusterFeatures() []FeatureName {
 		VirtualClusterProDistroFips,
 		VirtualClusterProDistroIntegrationsKubeVirt,
 		VirtualClusterProDistroIntegrationsExternalSecrets,
+		VirtualClusterConnectorExternalDatabase,
```

Comment:
I would call that `VirtualClusterSharedExternalDatabase` like Denise is using when talking about that feature

Reply:
No, we want it to be called connector as it's technically implemented as a connector.